crates/uv-install-wheel/src/wheel.rs

```diff
@@ -21,7 +21,24 @@ use walkdir::WalkDir;
 use zip::write::FileOptions;
 use zip::ZipWriter;
 
-const LAUNCHER_MAGIC_NUMBER: [u8; 4] = [b'U', b'V', b'U', b'V'];
+/// The kind of trampoline.
+///
+/// See [`uv-trampoline::bounce::TrampolineKind`].
+enum LauncherKind {
+    /// The trampoline should execute itself, it's a zipped Python script.
+    Script,
+    /// The trampoline should just execute Python, it's a proxy Python executable.
+    Python,
+}
+
+impl LauncherKind {
+    const fn magic_number(&self) -> &'static [u8; 4] {
+        match self {
+            Self::Script => &[b'U', b'V', b'S', b'C'],
+            Self::Python => &[b'U', b'V', b'P', b'Y'],
```

Comment:
I believe you can just write `b"UVSC"` and `b"UVPY"` here. (Fun fact, byte string literals have type `&'static [u8; N]` where as string literals have type `&'static str`.)